qa/tasks/deepsea.py

```diff
@@ -868,6 +868,71 @@ def begin(self):
     def teardown(self):
         pass
 
+class State(DeepSea):
+
+    def __init__(self, ctx, config):
+        deepsea_ctx['logger_obj'] = log.getChild('state')
+        super(State, self).__init__(ctx, config)
+        # cast stage/state_orch value to str because it might be a number
+        self.state = str(self.config.get("state", ''))
+        # targets all machines if omitted
+        self.target = str(self.config.get("target", '*'))
+        if not self.state:
+            raise ConfigError(
+                "(state subtask) nothing to do. Specify a value for 'state'")
```

Comment:
change "Specify a value" to "Specify a non-empty value" since "" empty string is still a value.

Reply:
done